assets/main.js

```diff
@@ -9,7 +6,71 @@ class Book {
   }
 }
 
-document.addEventListener('DOMContentLoaded', UI.displayBooks());
+function getBooksFromLocal() {
+  let books;
+  if (localStorage.getItem('books') === null) {
+    books = [];
+  } else {
+    books = JSON.parse(localStorage.getItem('books'));
+  }
+
+  return books;
+}
+
+function addBookToLocal(book) {
+  const books = getBooksFromLocal();
+  books.push(book);
+
+  localStorage.setItem('books', JSON.stringify(books));
+}
+
+function removeBookFromLocal(title, author, pages) {
+  const books = getBooksFromLocal();
+
+  books.forEach((b, index) => {
+    if (b.title === title && b.author === author && b.pages === pages) {
+      books.splice(index, 1);
+    }
+  });
+
+  localStorage.setItem('books', JSON.stringify(books));
+}
+
+function addBook(book) {
+  const table = document.querySelector('#book-list');
+  const row = document.createElement('tr');
+
+  row.innerHTML = `
+    <td>${book.author}</td>
+    <td>${book.title}</td>
+    <td>${book.pages}</td>
+    <td><button class ="btn btn-info read">Unread</btn></td>
+    <td><button class ="btn btn-danger delete">X</btn></td>
+  `;
+
+  table.appendChild(row);
+}
```

Comment:
[REQUIRED] Here you could also use object destructuring on the `book` parameter, to avoid code repetition.
![image](https://user-images.githubusercontent.com/73254040/98421878-021f2480-2050-11eb-9d52-0929b6f56419.png)